app/assets/javascripts/mapknitter/Map.js

```diff
@@ -1,9 +1,10 @@
-
+(function() {
```

Comment:
This is prob causing the `Minitest::UnexpectedError:         Selenium::WebDriver::Error::JavascriptError: javascript error: Cannot read property '_layers' of undefined` error by causing `map` to be assigned (line 21 below) only at the end. Hm.

Reply:
OK, tried removing it. Let's see.